backend/django/core/views/api_annotate.py

```diff
@@ -33,6 +36,8 @@
 from core.utils.utils_model import check_and_trigger_model
 from core.utils.utils_redis import redis_serialize_data, redis_serialize_set
 
+model = SentenceTransformer("multi-qa-MiniLM-L6-cos-v1")
```

Comment:
Would you mind adding a comment to the code giving the source for this model and a link to documentation? Just so it will be easy to remember what this is in the future.